sdk/anomalydetector/Azure.AI.AnomalyDetector/tests/samples/Sample4_MultivariateDetect.cs

```diff
@@ -209,5 +213,42 @@ await foreach (ModelSnapshot x in model_list)
             return count;
         }
         #endregion
+
+        #region Snippet:DetectLastMultivariateAnomaly
+        private async Task<LastDetectionResult> detectLastAsync(AnomalyDetectorClient client, Guid model_id)
+        {
+            Console.WriteLine("Start detect...");
+
+            List<VariableValues> variables = new List<VariableValues>();
+            variables.Add(new VariableValues("variables_name1", new[] { "2021-01-01 00:00:00", "2021-01-01 01:00:00", "2021-01-01 02:00:00" }, new[] { 0.0f, 0.0f, 0.0f }));
+            variables.Add(new VariableValues("variables_name2", new[] { "2021-01-01 00:00:00", "2021-01-01 01:00:00", "2021-01-01 02:00:00" }, new[] { 0.0f, 0.0f, 0.0f }));
+
+            LastDetectionRequest lastDetectionRequest = new LastDetectionRequest(variables, 1);
+
+            try
+            {
+                Response<LastDetectionResult> response = await client.LastDetectAnomalyAsync(model_id, lastDetectionRequest).ConfigureAwait(false);
+                if (response.GetRawResponse().Status == 200)
+                {
+                    foreach (AnomalyState state in response.Value.Results)
+                    {
+                        Console.WriteLine(String.Format("timestamp: {}, isAnomaly: {}, score: {}.", state.Timestamp, state.Value.IsAnomaly, state.Value.Score));
+                    }
+                }
+
+                return response;
+            }
+            catch (RequestFailedException ex)
+            {
+                Console.WriteLine(String.Format("Last detection failed: {0}", ex.Message));
+                throw new Exception(ex.Message);
```

Comment:
```suggestion
                throw;
```

Rationale discussed in a comment for Sample 2